documentation/modules/overview/con-configuration-points-configmaps.adoc

```diff
@@ -0,0 +1,78 @@
+// This module is included in:
+//
+// assembly-overview.adoc
+
+[id="configuration-points-configmaps-{context}"]
+= Using ConfigMaps to add configuration 
+
+[role="_abstract"]
+Use `ConfigMap` resources to add configuration to your Strimzi deployment.
```

Comment:
I'm a bit worried that this will be a bit misleading since users sometimes try to put config map everywhere. Can we make this more clear that they can do it only in some specific places? You go into more detail later, but I'm afraid that some will not get that far after seeing this.

Reply:
Changed to _...resources to add specific configuration_

And later:

> ConfigMaps can **only** store configuration data related to the following:
> 
> * Logging
> * Metrics
> * External configuration for Kafka Connect connectors
> 
> **You can't use ConfigMaps for other areas of configuration.**